invisible_cities/detsim/light_tables_c_test.py

```diff
@@ -34,6 +34,7 @@ def test_LT_SiPM_optional_arguments(get_dfs):
 @given(xs=floats(min_value=-500, max_value=500),
        ys=floats(min_value=-500, max_value=500),
        sipm_indx=integers(min_value=0, max_value=1500))
+@settings(deadline=None, max_examples=100)
```

Comment:
can we define `few_examples = settings(deadline=None, max_examples=100)`? Also I think it makes sense for `given` to be the closest decorator to the function definition.
Also, how bad was the deadline issue?